src/sortStudents.ts

```diff
@@ -1,16 +1,79 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Student[], sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  // Create a new array to hold the sorted students
+  const sortedStudents = [...students];
```

Comment:
```suggestion
  const studentsCopy = [...students];
```
Actually, they are not sorted, just a copy